src/core/service/notifications/jobs/handler.go

```diff
@@ -66,6 +68,8 @@ func (h *Handler) Prepare() {
 		return
 	}
 	h.status = status
+
+	h.executionMgr, _ = execution.NewDefaultManager()
```

Comment:
Use the `OperationController` defined here https://github.com/goharbor/harbor/blob/replication_ng/src/replication/ng/replication.go#L41. It's the interface for the upper level. If it contains no methods you need, just add them.

Reply:
done.